### DISCUSSION TITLE
Cesium ion as a Catalog Groups?

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#6311] [Ideas] [answered]
So I have lots of 3D data in my Cesium ion account. I thought it would be nice I could bring all assets from my account instead of registering as an indivisual catalog item.
Just a thought.

Thank you!
Hiroo

**Comment by nf-s** [comment, marked as answer]
Made an issue!
https://github.com/TerriaJS/terriajs/issues/6388

**Comment by imakihi** [comment]
Thank you, @nf-s and @na9da !!
